app/src/components/buttons/network-list-button.tsx

```diff
@@ -19,11 +22,20 @@ interface Props {
 
 export const NetworkListButton = (props: Props) => {
   const { network, onClick } = props
+  // Hooks
+  const { getNetworkFeeFiatEstimate } = useNetworkFees()
 
+  // Methods
   const onSelectNetwork = React.useCallback(() => {
     onClick(network)
   }, [network, onClick])
 
+  // Memos
+  const networkFeeFiatValue = React.useMemo(() => {
+    const fee = Number(getNetworkFeeFiatEstimate(network)).toFixed(4)
+    return fee ? `$${fee}` : ''
```

Comment:
Maybe create a follow-up to use the selected fiat currency symbol

Reply:
Issue created with a comment! https://github.com/brave/brave-browser/issues/25262